helix-core/src/test/java/org/apache/helix/integration/task/TestEnqueueJobs.java

```diff
@@ -231,12 +272,14 @@ public void testQueueJobsMaxCapacity() throws InterruptedException {
 
     _driver.resume(queueName);
 
+    System.out.println("before the polls");
```

Comment:
Please check the whole PR and remove these prints is possible.

Reply:
removed.